docs/platforms/javascript/common/configuration/integrations/browserapierrors.mdx

```diff
@@ -32,6 +32,7 @@ This integration wraps native time and event APIs (`setTimeout`, `setInterval`,
 
 ```JavaScript
 Sentry.init({
+  dsn: "___PUBLIC_DSN___",
```

Comment:
I would honestly not add the DSN here (and all the other snippets). It is not relevant to the snippet and it will create an ugly banner on top of the snippet if you are not logged in.

Reply:
yeah let's go for nothing - I agree!